From 66d8772f0217fb4fa4ab90c090cef12263d8282f Mon Sep 17 00:00:00 2001
From: Sultanxda <sultanxda@gmail.com>
Date: Thu, 20 Aug 2015 20:06:08 -0700
Subject: [PATCH 046/194] SnapdragonCamera: Fix incorrect viewfinder ratio for
 13.1MP shots

Set the aspect tolerance to 0.02 to compensate for 13.1MP (4208x3120) shots. 13.1MP will now be properly registered as a 4/3 resolution when the viewfinder ratio is calculated.

Change-Id: I41540a0d5fbe11f59d776a6e70e82baa463fff59
---
 src/com/android/camera/util/CameraUtil.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/camera/util/CameraUtil.java b/src/com/android/camera/util/CameraUtil.java
index 78addeced..1b236e17f 100755
--- a/src/com/android/camera/util/CameraUtil.java
+++ b/src/com/android/camera/util/CameraUtil.java
@@ -607,7 +607,7 @@ public class CameraUtil {
         // TODO(andyhuibers): Don't hardcode this but use device's measurements.
         final int MAX_ASPECT_HEIGHT = 1080;
         // Use a very small tolerance because we want an exact match.
-        final double ASPECT_TOLERANCE = 0.01;
+        final double ASPECT_TOLERANCE = 0.02;
         if (sizes == null) return -1;
 
         int optimalSizeIndex = -1;
-- 
2.17.1

